src/features/txnDelegation/useSolTxns.ts

```diff
@@ -139,6 +144,22 @@ const useSolTxns = (heliumAddress: string, solanaTransactions?: string) => {
     [],
   )
 
+  const assetToAddress = useCallback(
```

Comment:
There's a function in `helium-entity-manager-sdk` that does this. Should use that so we don't have to maintain this gross logic in multiple places

Reply:
@matthewcarlreetz bump on this, looks like this feature is needed

Reply:
@ChewingGlass what function are you referring to?

Reply:
I'm already using `import { decodeEntityKey } from '@helium/helium-entity-manager-sdk'`